node/osfamilymap.yaml

```diff
@@ -28,7 +28,7 @@ Debian:
             {%- if 'oscodename' in grains %}
     repo:
       humanname: NodeSource Node.js Repository
-      name: deb https://deb.nodesource.com/node_13.x {{ grains.oscodename }} main
```

Comment:
I agree with this PR.  Could you add `node:pkg:version: '14'` to `defaults.yaml` so formula works without pillars?

Reply:
It should pick version from default.yaml unless (optional) pillars say different version.

Reply:
changed, thank you